lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -1186,8 +1184,6 @@ bool lldb_private::formatters::swift::GLKit_SummaryProvider(
   if (!process_sp)
     return false;
```

Comment:
what about this condition? Or is process_sp used later on?

Reply:
It's not used later on. It seems safe to delete to me. Is there ever a case where a value would be formatted without an associated process?

Reply:
Let's get rid of it!

Reply:
done